src/scrapers/facebook.iced

```diff
@@ -108,12 +110,3 @@ exports.FacebookScraper = class FacebookScraper extends BaseScraper
       if proof_text_check.split(/\s+/).indexOf(med_id)  < 0
```

Comment:
Reading your comment this sounded like a good idea, but now that I look at it I don't think I can. The `validate` function on scrapers (which calls `_validate_text_check`) is actually unused right now, because our only caller for this code is the server, which has no reason to validate the hint that it just generated itself. (Instead it calls `check_status` directly.) We _used_ to have a caller in the node client before that died, and you can see the arguments we used to pass into `validate` [here](https://github.com/keybase/node-client/blob/master/src/chainlink.iced#L242-L260). It looks like we never actually supplied the Keybase username to this function :p